src/Popover.js

```diff
@@ -3,6 +3,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import isRequiredForA11y from 'prop-types-extra/lib/isRequiredForA11y';
 import { createBootstrapComponent } from './ThemeProvider';
+import PopoverTitle from './PopoverTitle';
+import PopoverContent from './PopoverContent';
```

Comment:
could be useful to have something like a `<Popover content>` shorthand like `<Card body>`: https://react-bootstrap.github.io/components/cards/#body

Reply:
added this in 9216e840ba273a3a1ec44bd8e179b5d545f73877